lib/ansible/modules/apt.py

```diff
@@ -356,21 +356,21 @@ def __init__(self, module):
 
     def __enter__(self):
         """
-        This method will be call when we enter the context, before we call `apt-get …`
+        This method will be called when we enter the context, before we call `apt-get …`
         """
 
         # if policy_rc_d is null then we don't need to modify policy-rc.d
         if self.m.params['policy_rc_d'] is None:
             return
 
-        # if the /usr/sbin/policy-rc.d already exist we back it up
+        # if the /usr/sbin/policy-rc.d already exists we back it up
         if self.backup_dir:
             try:
                 shutil.move('/usr/sbin/policy-rc.d', self.backup_dir)
             except Exception:
                 self.m.fail_json(msg="Fail to move /usr/sbin/policy-rc.d to %s" % self.backup_dir)
 
-        # we write /usr/sbin/policy-rc.d so it always exit with code policy_rc_d
+        # we write /usr/sbin/policy-rc.d so it always exits with code policy_rc_d
```

Comment:
I'm not 100% sure on this one...seems like the intention could have been "exists", so I can update this PR if that is the intended meaning here.

Reply:
@lpmi-13 thanks for the PR. This looks correct to me - `fail_json` will exit the code.